content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -48,10 +48,14 @@ simultaneously are also compatible.
 
 ## Access the Namespace on Cluster/Project of the Running CI Job
 
+It is possible to authenticate to CI build farms with GitHub OAuth as well as Red Hat Single-Sign-On: GitHub Oauth requires that you are a member
+of the `OpenShift` organization. Both the authors of a pull request and the corresponding Red Hat kerberos IDs are permitted to accessing the
+`Namespace` that run jobs.
+
 {{< alert title="Info" color="info" >}}
-While it is possible to authenticate to CI build farms with GitHub OAuth as well as Red Hat Single-Sign-On, accessing
-`Namespaces` that run jobs is only supported using the GitHub authentication method and requires that you are a member
-of the `OpenShift` organization.
+Under the hood, the mapping `m(github_login)=kerberos_id` is stored for each Red Had user that sets up *GITHUB* at *PROFESSIONAL SOCIAL MEDIA* with Rover People.
+For each `github_login`, there is a group `github_login-group` on CI build farms containing both `github_login` and `kerberos_id`. The group `github_login-group` has admin role in the `Namespace` that run jobs.
+The users who do not set this up have no the permission and it takes one day to synchronize the modification at Rover People to the build farms.
```

Comment:
The paragraph is IMO a little chaotic and does not read very well, especially from user point of view. They do not care about under the hood groups and mappings.

"OpenShift CI obtains the correspondence between GitHub logins and Red Hat logins using the *PROFESSIONAL*... yada yada yada in Rover. It may take one day for changes in Rover profile to propagate to OpenShift CI. Users who do not have this information in Rover may only access the `Namespace` using the GitHub OAuth login.

Reply:
> Users who do not have this information in Rover may only access the Namespace using the GitHub OAuth login.

We lost this with the current implementation because
`ci-op` use the group for rolebinding and the group exist only for the users who set up GitHub on Rover.

Reply:
The options I can think of:
- `ci-op` could check if the <github_login>-group exist. Use group in the rolebinding if it does. Use the user as before if not.
- `group-updater` maintains the groups for all github users that are not in the mapping.

We could make `configuring` GITHUB at Rover as a requirement of accessing the Namespace. If I recall, DPP has announced something like this. Then we do not need to change our code.

Reply:
Yeah, I don't think we need to bother supporting people with GH logins only. It's fine. Let's just document the current behavior and that's it.

Very soon we will not GH auth at all, so that use case will go away anyway.